aws/platform/variables.tf

```diff
@@ -211,6 +211,10 @@ variable "prometheus_data_source" {
     # The write path for the Prometheus workspace. 
     # A sample value for AWs managed Prometheus will be `workspaces/ws-xxxxx-xxx-xxx-xxx-xxxxxxx/api/v1/remote_write`
     write_path = string
+
+    # The url for the Prometheus workspace. 
+    # A sample value for AWs managed Prometheus will be `https://aps-workspaces.us-east-1.amazonaws.com/workspaces/ws-xxxxx-xxx-xxx-xxx-xxxxxxx`
+    url = string
```

Comment:
The pull request title mentions `name`, but this is adding `url`. Is that intentional?

Reply:
Both `name` and `url` were being added for different reasons. Since `name` is already available from recent commits, I will amend to the PR to show only the `url` is being updated.